pkgs/by-name/du/dunst/package.nix

```diff
@@ -102,16 +105,20 @@ stdenv.mkDerivation (finalAttrs: {
       --replace-fail "jq" "${lib.getExe jq}"
   '';
 
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  versionCheckProgramArg = [ "--version" ];
+  doInstallCheck = true;
+
   passthru = {
-    tests.version = testers.testVersion {
-      package = finalAttrs.finalPackage;
-    };
     updateScript = nix-update-script { };
   };
 
   meta = {
     description = "Lightweight and customizable notification daemon";
     homepage = "https://dunst-project.org/";
+    changelog = "https://github.com/dunst-project/dunst/blob/${finalAttrs.src.tag}/CHANGELOG.md";
```

Comment:
Using `tag` breaks if a user overrides the src using `rev` https://github.com/nix-community/nixpkgs-wayland/actions/runs/12472468718/job/34811407386#step:5:3478